site/en/codelabs/openthread-network-simulator/index.lab.md

```diff
@@ -88,75 +83,53 @@ $ git clone https://github.com/openthread/ot-ns.git ./otns
 $ cd otns
 ```
 
-### Install Dependencies
-
-```console
-$ ./script/install-deps
-grpcwebproxy installed: /usr/local/google/home/simonlin/go/bin/grpcwebproxy
-```
-
-You might be asked to input a password for `sudo`.
-
-### **Install otns**
-
-Install `otns` to `$GOPATH/bin`:
-
-```console
-$ ./script/install
-otns installed: /usr/local/google/home/simonlin/go/bin/otns
-```
-
-#### Let's check if `otns` is properly installed
+All subsequent console commands in this Codelab are run from the `otns` directory. 
 
-1. Run `which otns` to check if the `otns` executable is searchable in `$PATH.`
-2. If the `otns` command is not found, verify that you have added
-   `$(go env GOPATH)/bin` to `$PATH.`
+### Bootstrap and Install
 
-
-## Build OpenThread for OTNS
-Duration: 05:00
-
-
-### Get OpenThread code from GitHub
+The `bootstrap` script will install dependencies (including Python3 and Go/Golang, if needed) and install OTNS.
+It also builds the various OT node types that can be used directly in a simulation, and it performs some basic tests.
+Due to the node builds, it can take several minutes.
 
 ```console
-$ mkdir -p ~/src
-$ git clone https://github.com/openthread/openthread ~/src/openthread
+$ ./script/bootstrap
+....
+....
+OTNS installed - use 'otns' to start it.
+$
 ```
 
-### Build OpenThread with `OTNS=1`
+You might be asked to input a password for `sudo`.
 
-```console
-$ cd ~/src/openthread
-$ ./script/cmake-build simulation -DOT_OTNS=ON -DOT_SIMULATION_VIRTUAL_TIME=ON -DOT_SIMULATION_VIRTUAL_TIME_UART=ON -DOT_SIMULATION_MAX_NETWORK_SIZE=999
-```
+### If `otns` is not properly installed
 
-You can find the OpenThread executables in the `build` directory:
+The script may report an error like:
 
 ```console
-$ ls ~/src/openthread/build/simulation/examples/apps/cli/
-ot-cli-ftd        ot-cli-mtd        ot-cli-radio
+....
+OTNS installed - please add ~/go/bin to your PATH variable first, to use it.
+$
 ```
 
-Now it's time to run OTNS...
+In this case, you need to add `$(go env GOPATH)/bin` to your `$PATH.`
 
+In case of other errors, a [Github issue](https://github.com/openthread/ot-ns/issues) can be created.
 
-## Run OTNS
+## Run OTNS For the First Time
 Duration: 01:00
 
 
 Run `otns`:
 
 ```console
-$ cd ~/src/openthread/build/simulation/examples/apps/cli
 $ otns
-> ← OTNS-CLI prompt
+>_ ← OTNS-CLI prompt
 ```
 
 When successfully started, OTNS will enter a CLI console (`OTNS-CLI`) and
 launch a web browser for network visualization and management (`OTNS-Web`):
 
-<img src="img/a0e05178d66929b1.png" alt="a0e05178d66929b1.png" width="624.00" />
+<img src="img/00_otns_web.png" alt="OTNS-Web window at start" width="624.00" />
 
 **If you can only see a blank page for OTNS-Web, chances are WebGL is not
 enabled in your browser. Please refer to**
```

Comment:
Can you clean up this paragraph while you're at it, please.  Should just be a single bold around the whole thing, and needs a space between "to" and the link.